en/docs/learn/enterprise-integrations-patterns.md

```diff
@@ -54,6 +59,11 @@ The WSO2 Ballerina Integrator supports the implementation of key Enterprise Inte
 ## Message construction
 
 <table>
+    <tr>
+        <td><a href="https://github.com/wso2/integration-samples/tree/main/EI%20patterns/command_message" target="_blank">Command Message</a></td>
+        <td><a href="{{base_path}}assets/img/learn/eip/command-message.svg"><img src="{{base_path}}/assets/img/learn/eip/command-message.svg" alt="command-message" width="80"></a></td>
+        <td>How to use message to invoke a procedure in another application</td>
```

Comment:
Please use the same description in https://www.enterpriseintegrationpatterns.com/patterns/messaging/toc.html. Refer to https://mi.docs.wso2.com/en/latest/learn/enterprise-integration-patterns/eip-overview/ as well (it is usign the same.)